packages/consumption/src/modules/attributes/local/LocalAttributeDeletionInfo.ts

```diff
@@ -11,21 +11,22 @@ export enum DeletionStatus {
 }
 
 export interface LocalAttributeDeletionInfoJSON {
-    deletionStatus: string;
+    deletionStatus: LocalAttributeDeletionInfoStatus;
```

Comment:
actually Enums in ts are just strings in js.

Reply:
Sure, just wondering because `ThirdPartyRelationshipAttributeQueryOwner` is an `enum` too and we decided not to use it as the type for the `owner` of the ThirdPartyRelationshipAttributeQueryJSON, but as the type for the `owner` of the IThirdPartyRelationshipAttributeQuery. I think there are more examples in our code where we use this approach to typing, just as we used string for the `deletionStatus` property as type of the `LocalAttributeDeletionInfoJSON` before.

Reply:
fixed it now also for ThirdPartyRelationshipAttributeQueryOwner

Reply:
> ThirdPartyRelationshipAttributeQueryOwner

❤️‍🔥

Reply:
what's this emote? 🗡️

Reply:
aah, heart on fire :D